systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/ResourceManager.java

```diff
@@ -5,33 +5,24 @@
 package io.enmasse.systemtest.messaginginfra;
 
 import io.enmasse.address.model.AddressSpace;
-import io.enmasse.api.model.MessagingInfra;
-import io.enmasse.api.model.MessagingInfraBuilder;
 import io.enmasse.iot.model.v1.IoTProject;
 import io.enmasse.systemtest.Environment;
 import io.enmasse.systemtest.UserCredentials;
 import io.enmasse.systemtest.amqp.AmqpClientFactory;
 import io.enmasse.systemtest.bases.ThrowableRunner;
 import io.enmasse.systemtest.logs.CustomLogger;
 import io.enmasse.systemtest.logs.GlobalLogCollector;
-import io.enmasse.systemtest.messaginginfra.crds.MessagingInfraCrd;
 import io.enmasse.systemtest.mqtt.MqttClientFactory;
 import io.enmasse.systemtest.platform.Kubernetes;
-import io.enmasse.systemtest.time.TimeoutBudget;
 import io.fabric8.kubernetes.api.model.HasMetadata;
-import io.fabric8.kubernetes.client.dsl.MixedOperation;
 import org.slf4j.Logger;
 
-import java.time.Duration;
 import java.util.Stack;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
 /**
  * Managing resources //TODO rename it once enmasse will be moved to 1.0
  */
-public class MessagingInfraResourceManager {
+public class ResourceManager {
```

Comment:
@lulf nice would be create universal method getResource(ResourceType type, String namespace, String name) ?

Reply:
I was thinking about that originally, but then I thought about the scope of ResourceManager being to manage lifecycle of test resources, not as a 'Kubernetes API' for tests, so I did not include it.

Using the ResourceType directly as I've done in the tests seemed simpler to me.

Reply:
agreed you are right